### PR TITLE
Remove discoveryManager

### PR DESCRIPTION
**Pull request by rmetzger**
(do not merge immediately -- not tested with yarn. (should not make any difference) )

Fix for issue: https://github.com/stratosphere/stratosphere/issues/18

I removed the DiscoveryManager, as it is not really used.

The detection of the TaskManager's own IP address does not need a RPC service at the JobManager. It is determined based on the network adapter that successfully connects to the JobManager.

Please review my implementation.

### CODE REVIEW

File: stratosphere-dist/src/main/stratosphere-bin/bin/start-cluster.sh [hunk 1]
```diff
@@ -39,6 +39,6 @@ do
     read line || GOON=false
     if [ -n "$line" ]; then
         HOST=$( extractHostName $line)
-        ssh -n $NEPHELE_SSH_OPTS $HOST -- "nohup /bin/bash $NEPHELE_BIN_DIR/nephele-taskmanager.sh start &"
+        ssh -n $NEPHELE_SSH_OPTS $HOST -- "nohup /bin/bash $NEPHELE_BIN_DIR/nephele-taskmanager.sh start &" &
```
Comment (StephanEwen) [inline stratosphere-dist/src/main/stratosphere-bin/bin/start-cluster.sh]: Did you check that this works properly? Do we still see error messages if the remote command failed?

**Comment by StephanEwen** [comment]
Ready to be merged?

**Comment by rmetzger** [comment]
No. I'll take a quick look on the start-cluster.sh script.

On Wed, Dec 4, 2013 at 4:39 PM, Stephan Ewen notifications@github.comwrote:

> Ready to be merged?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/stratosphere/stratosphere/pull/313#issuecomment-29815206
> .

## 

Robert Metzger, Kontakt: metzgerr@web.de, Mobil: 0171/7424461

**Comment by rmetzger** [comment]
I removed the start-cluster.sh script changes, so ready to merge.

**Comment by StephanEwen** [comment]
Merged in a6cbd9408ee7c5859d7ba0599c2554551bceca80

**Pull request closed without merging**